src/test/java/core/basesyntax/handler/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.handler;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationHandlerTest {
+    private static Fruit apple;
+    private static OperationHandler balanceOperationHandler;
+    private static FruitTransaction fruitTransaction;
+
+    @BeforeAll
+    static void setUp() {
+        apple = new Fruit("apple");
+        balanceOperationHandler = new BalanceOperationHandler();

```

Comment:
StorageDao in your implementations must be initialized in the constructor. Same everywhere